Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/non_rigid_mesh_registration.h

```diff
@@ -0,0 +1,758 @@
+// Copyright (c) 2024 GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Roberto Dyke, Sven Oesau
+//
+#ifndef CGAL_POLYGON_MESH_PROCESSING_REGISTER_MESH_H
+#define CGAL_POLYGON_MESH_PROCESSING_REGISTER_MESH_H
+
+#include <CGAL/license/Polygon_mesh_processing/miscellaneous.h>
```

Comment:
@sloriot do we need a new category?

Reply:
it's not yet clear to me that it shouldn't be in its own package. But to begin with you might want to create a new license group.